modules/@apostrophecms/schema/ui/apos/components/AposInputArea.vue

```diff
@@ -19,7 +19,7 @@
 
 <script>
 
-import AposInputMixin from '../mixins/AposInputMixin.js';
+import AposInputMixin from 'Modules/@apostrophecms/schema/mixins/AposInputMixin';
```

Comment:
I used this syntax to reference this file in the new field component, so it seemed good to switch to this everywhere.

Reply:
👍